src/operator/tensor/elemwise_unary_op.cc

```diff
@@ -454,10 +436,7 @@ MXNET_OPERATOR_REGISTER_BINARY(_backward_radians)
 
 // sinh
 MXNET_OPERATOR_REGISTER_UNARY(sinh)
-.describe(R"code(Hyperbolic sine, element-wise.
-
-For example::
-   sinh(x) = 0.5\times(exp(x) - exp(-x))
+.describe(R"code(Returns element-wise Hyperbolic sine value of the input.
```

Comment:
i still feel the math eq explains things. otherwise a user may need to google the definition of sinh